tests/framework/results/helpers.py

```diff
@@ -92,51 +92,61 @@ def on_time_step(self, pop_data: SimulantData) -> None:
 
 
 class HousePointsObserver(Component):
+    """Observer that is stratified by multiple columns (the defaults,
+    'student_house' and 'power_level')
+    """
+
     def setup(self, builder: Builder) -> None:
         builder.results.register_observation(
             name="house_points",
             aggregator_sources=["house_points"],
             aggregator=sum,
             requires_columns=[
                 "house_points",
-                "student_house",
-                "power_level",
+                # "student_house",
+                # "power_level",
             ],
         )
 
 
 class FullyFilteredHousePointsObserver(Component):
+    """Same as `HousePointsObserver but with a filter that leaves no simulants"""
+
     def setup(self, builder: Builder) -> None:
         builder.results.register_observation(
             name="house_points",
             pop_filter="tracked==True & power_level=='one billion'",
             aggregator_sources=["house_points"],
             aggregator=sum,
-            requires_columns=[
-                "house_points",
-                "student_house",
-                "power_level",
-            ],
+            # requires_columns=[
+            #     "house_points",
+            #     "student_house",
+            #     "power_level",
+            # ],
         )
 
 
 class QuidditchWinsObserver(Component):
+    """Observer that is stratified by a single column ('familiar')"""
+
     def setup(self, builder: Builder) -> None:
         builder.results.register_observation(
             name="quidditch_wins",
             aggregator_sources=["quidditch_wins"],
             aggregator=sum,
-            excluded_stratifications=["student_house"],
+            excluded_stratifications=["student_house", "power_level"],
```

Comment:
I'm not the biggest Harry Potter fan but I don't remember anyone having a power level attribute.

Reply:
You'll have to ask Matt about this